### ISSUE TITLE
MJPEG port is not forwarded to localport

### ISSUE THREAD

**Issue by PrathyushaLam** [appium/appium-xcuitest-driver#1316]
started session with mjpeg port included in capabilities.

session started
wda port can be pinged with localhost:8100
mjpeg port not working with localhost:9100/ but working with device_ip:9100/

possible to fix port forward for mjpeg port as well.

**Comment by jlipps** [comment]
I don't think this is possible, Apple doesn't give us a way to forward ports from their devices. Can you confirm @mykola-mokhnach?

**Comment by PrathyushaLam** [comment]
similar to forwarding wda port 8100 using iproxy from usbmuxd. Why not forward port of mjpeg(9100) also to local?

**Comment by jlipps** [comment]
yeah good point. have you tried using iproxy on your own? does it work?

**Comment by PrathyushaLam** [comment]
yes, i tried and it works. but why not integrate it with appium-xcuitest-driver. That is the reason i raised the issue

**Comment by jlipps** [comment]
I see. Now I understand you. We can probably provide a capability for this in the real device case.

**Comment by KazuCocoa** [comment]
It seems by design when the feature was implemented. The PR expected to use iproxy for real devices if a user wanted to forward them.
Screen recording feature does port forward the 9100 port by default in https://github.com/appium/appium-xcuitest-driver/blob/5675b01cb18b252e5e8d1688bb51844d47d290ea/lib/commands/recordscreen.js,  So probably we should coordinate the implementation as well.

**Comment by mykola-mokhnach** [comment]
Yes, it is by design. We only request port forward when screen recording starts and stop it after it stops. I don't see a reason to change that behaviour.
